LibertyDSNP/sdk-ts
# Use parquetjs v1.1.0

Problem
=======
[Related to Pivotal Tracker #178456668](https://www.pivotaltracker.com/story/show/178456668), Now that parquetjs has a 1.1.0 release, point the SDK at it.

Solution
========
Update package*.json

Steps to Verify:
----------------
1.  CI should pass
1.  The SDK should be useable by other repos now that LZO support is gone completely.

Not included:
---
To get a browser version of the SDK we'll need to add a browser target similar to what we did for [parquetjs v1.1.1](https://github.com/LibertyDSNP/parquetjs/pull/17)
